monitoring/scripts/populate_dummy_calm_vhs.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8
+
+import os
+import json
+import boto3
+import hashlib
+from download_oai_harvest import fetch_calm_records
```

Comment:
nit: PEP 8 import order would split out `boto3` and `download_oai_harvest` as separate import groups, because they're third-party and first-party respectively.